src/app/developer/components/Home.spec.tsx

```diff
@@ -16,7 +16,7 @@ describe('Home', () => {
 
   it('matches snapshot', async() => {
     jest.spyOn(Date.prototype, 'getFullYear').mockReturnValue(2021);
-    const store = createTestStore({navigation: new URL('https://localhost')});
+    const store = createTestStore({navigation: new URL('https://localhost') as any});
```

Comment:
why is this necessary?

Reply:
```
Type 'URL' is not assignable to type 'State'.
  Property 'state' is missing in type 'URL' but required in type 'Location<any>'.ts(2322)
  ```
  I guess this didn't change recently but now it is an error so it should be okay to just use `any`

Reply:
if url legitimately doesn't have a field that state requires, it would probably be better to add that field by doing
```
{...new URL('https://localhost'), state: {}}
```
or whatever